### ISSUE TITLE
Training the voxel net for different objects at a time?

### ISSUE THREAD

**Issue by GuidanceMantras** [qianguih/voxelnet#90]
Hey,

training the model for different models at a time?

Kindly let me know

Thanks in advance

**Comment by Manueljohnson063** [comment]
Hai,I am stuck with this in adding multiple objects in training could you please help how to add multiple objects!!!!

**Comment by GuidanceMantras** [comment]
@Manueljohnson063  

Sorry for the late. I am also stuck there only. I am trying to add multiple objects for training, but so far no success

**Comment by GuidanceMantras** [comment]
Hi @Manueljohnson063

Here is the solution to add multiple objects to voxelnet.  Still it is not efficient one, but it works. So you have to find the cls_loss_pedestrian and reg_loss_pedestrian as well as for cyclist and car for each object individually and add them to find the final cls_loss and reg_loss. Here one has to use hyper parameters to balance the classification loss. 

Since most of the frame have more cars than pedestrian, and also most of the frame has more pedestrian than cyclist. It is upto the user how he/she can balance this using hyper parameters.

Another solution would be similar to the above one, but with different loss function using one hot encoding! Since I am doing my master thesis, I do not want to take any risk in trying other solution(efficient one), when existing solution already works. 

Hope this helps you! 

![mulit_classification](https://user-images.githubusercontent.com/53983084/111427708-c92a4700-86f6-11eb-84bc-738ada825183.png)

**Comment by sofiaesp** [comment]
@Suri12990 How do you modify model to train for cyclist and pedestrian, and how do you add the loss? I would appreciate your help.

**Comment by sofiaesp** [comment]
Hi I found that you can modify the selected object in config.py, I have to train a model for each object? If so, when I get the three models (one for each object) how could I add them to find a final loss? @Suri12990

**Comment by gynaa** [comment]
hi @sofiaesp did  you figure it out? i would really appreciate your help
